services/kubecost/metadata.yaml

```diff
@@ -30,7 +30,7 @@ overview: |-
 
   ## More Information
   - [kubecost.com](https://www.kubecost.com/)
-  - [Centralized Cost Monitoring](https://docs.d2iq.com/dkp/kommander/latest/monitoring/centralized-cost-monitoring/)
+  - [Centralized Cost Monitoring](https://docs.d2iq.com/dkp/latest/centralized-cost-monitoring)
```

Comment:
what if a user on 2.4 gets sent to our docs for 2.6 sometime in the future? Is that what we want?

Reply:
Very good question and our intent of the old URL was just that. Not sure if things changed on that end.

Reply:
New approach is to pin, @msdolbey  made clear to me how important that change is.

We already landed the 2.3 release branch version, pointing to `2.3`; https://github.com/mesosphere/kommander-applications/pull/504

By now using `2.4` here, we are carrying forward the intent to pin this URL. that in turn means we will have to update that one with every major release.